lib/plugins/repository.js

```diff
@@ -252,4 +256,39 @@ module.exports = class Repository {
       }
     }
   }
-}
+
+  updateAutomatedSecurityFixes(repoData, resArray) {
+    if (this.security) {
+      if (this.security?.enableAutomatedSecurityFixes === true || this.security?.enableAutomatedSecurityFixes === false) {
+        if (this.security.enableAutomatedSecurityFixes === true) {
+          this.log(`Enabling Dependabot security updates for owner: ${repoData.owner.login} and repo ${repoData.name}`)
+          if (this.nop) {
+            resArray.push((new NopCommand(this.constructor.name, this.repo, this.github.repos.enableAutomatedSecurityFixes.endpoint({
+              owner: repoData.owner.login,
+              repo: repoData.name
+            }), 'Update Topics')))
```

Comment:
I guess this is a copy&paste error. This call has nothing to do with topics.

Reply:
@martinm82 yeap, and it seems to be coming from `updateSecurity` as well. I'll go ahead and update both.